msal/application.py

```diff
@@ -1224,6 +1225,12 @@ def acquire_token_silent(
             returned from the UserInfo Endpoint and/or in the ID Token and/or Access Token.
             It is a string of a JSON object which contains lists of claims being requested from these locations.
 
+        :param object auth_scheme:
```

Comment:
Is there public API similar to `IsPOPSupportedByClient` as per spec?

Reply:
1. Currently, no. And we can easily add a new `is_broker_enabled()` public API, whose name was derived from [the specs](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=/PoPTokensProtocol/PoP_API_In_MSAL.md&_a=preview&anchor=proposal-2---optional-isproofofposessionsupportedbyclient-helper-(accepted)) which mentioned:

   > Note: internally, MSAL java, py, .NET and js would implement this as `public bool IsBrokerAvailable()`.

2. There is a follow-up question. Does such a new `is_broker_enabled()` public API have any implication to the debatable ["naming of `allow_broker` VS `enable_broker`" topic](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/596#discussion_r1357920721)? Now this new `is_broker_enabled()` helper indeed makes it obvious that "even after opting in `enable_broker_on_windows=True`, there is still no guarantee that the broker is actually enabled". Then we shall probably name the opt-in flag as `allow_broker_on_windows=True`.

Reply:
`is_pop_supported_by_client` seems like a better feature flag, because it hides the implementation detail of `is_broker_available`.

Reply:
Based on offline discussion, we agreed on a short `PCA.is_pop_supported()`. It accepts no `scope` parameter thus implies that it is about client's POP readiness.